Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN apk --no-cache add ca-certificates
 COPY --from=builder /nkn/nknd /nkn/
 COPY --from=builder /nkn/nknc /nkn/
 COPY --from=builder /nkn/config.mainnet.json /nkn/
+ADD ./certs /nkn
```

Comment:
We should use `COPY --from=builder /nkn/certs /nkn/` because certs is already in builder (1st stage image)

Reply:
done